dace/codegen/targets/rtl.py

```diff
@@ -548,19 +592,20 @@ def unparse_tasklet(self, sdfg: sdfg.SDFG, dfg: state.StateSubgraphView, state_i
                 raise NotImplementedError('Only RTL codegen for Xilinx is implemented')
         else:  # not hardware_target
             # generate verilator simulation cpp code components
-            inputs, outputs = self.generate_cpp_inputs_outputs(tasklet)
-            valid_zeros, ready_zeros = self.generate_cpp_zero_inits(tasklet)
+            inputs, outputs = self.generate_cpp_inputs_outputs(
+                tasklet, buses)  # TODO i don't know if i want to do it differently.
```

Comment:
Elaborate!

Reply:
It was a comment to an old way of doing things. This new version uses buses, which already contains all of the collected information needed.